bundles/org.openhab.binding.ephemeris/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ephemeris"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="file">
+		<label>Ephemeris File</label>
+		<description>Events defined in a Jollyday file</description>
+
+		<channels>
+			<channel id="title-today" typeId="event_current_title"/>
+			<channel id="event-today" typeId="in-dayset">
+				<label>Event Today</label>
+				<description>Set to ON if an event exists today</description>
+			</channel>
+			<channel id="event-tomorrow" typeId="in-dayset">
+				<label>Event Tomorrow</label>
+				<description>Set to ON if an event exists tomorrow</description>
+			</channel>
+			<channel id="next-title" typeId="event_next_title"/>
+			<channel id="next-start" typeId="event_next_start"/>
+			<channel id="days-remaining" typeId="remaining_days"/>
```

Comment:
It looks just strange to have "days-remaining" and "remaining-days". You should probably use only one of them.

Reply:
Well spotted.